src/index.js

```diff
@@ -166,13 +80,13 @@ class UglifyJsPlugin {
       }
 
       compilation.plugin('optimize-chunk-assets', (chunks, callback) => {
+        const uglify = new Uglify(this.options.parallel);
```

Comment:
Latest commit: `uglify/index.js` handles this logic, it keeps the plugin's original workflow :)